subscription-engine/kafka-ws/docs/apiary.apib

```diff
@@ -6,15 +6,17 @@ FORMAT: 1A
 The Kafka WS service provides support for pure WebSocket connections to retrieve data directly
 from Kafka.
 
-# Group Ticket
+# Group ticket
```

Comment:
The word `Group` is actually a _marker_, it is not displayed, so why is the word **t**icket changed to lowercase while the next `Group` marker has the word **W**ebSocket in uppercase? Just my curiosity...